test/constants_test.go

```diff
@@ -16,7 +14,6 @@ const (
 
 	latencyAsync         = 100 * time.Millisecond
 	maxRounds            = 10
-	asyncIterations      = 5000
```

Comment:
> how this is doing anything fundamentally different.

It doesn't. It just gives us the same sort of coverage with coverage guided fuzzing, where the tuning knob is fuzz time instead of a fixed number of iterations.

>  I need convincing that we've done that

Would a before/after test coverage report suffice?

Reply:
Only a little. Unfortunately I don't think line coverage is a super great metric. We need internal state space coverage.

Reply:
> We need internal state space coverage.

Do you have suggestions on how I can measure the internal state space coverage compared to fuzz testing? 
Would replicating what tests do in fuzz tests be sufficient to replace test repetitions with fuzz tests in your oppinion?

Reply:
Yes, replicating is fine. I think finding a metric for state space coverage is probably beyond scope here, but something to think about. Possibly about as good would be a metric coverage of the permutations of the sequence of messages observed, which we might be able to measure in the message queue later.